Makefile

```diff
@@ -39,16 +39,16 @@ _output/bin/nerdctl.lima:
 
 .PHONY: _output/bin/limactl
 _output/bin/limactl:
-	$(GO_BUILD) -o $@ ./cmd/limactl
+	CGO_ENABLED=1 $(GO_BUILD) -o $@ ./cmd/limactl
```

Comment:
Please add a comment line to explain the reason

Reply:
cgo conflicts with cross-compilation.

I'd suggest shelling out nslookup/dig/getent or whatever (with caching the result